src/stories/SystemFields.stories.ts

```diff
@@ -0,0 +1,57 @@
+import type { Meta, StoryObj } from '@storybook/vue3';
+import SystemFields from '@components/smart/SystemFields.vue';
+
+const meta: Meta<typeof SystemFields> = {
+  title: 'Smart components/SystemFields',
+  component: SystemFields,
+  argTypes: {
+    status: { control: 'text' },
+    createdBy: { control: 'text' },
+    createdAt: { control: 'text' },
```

Comment:
Er bestaat ook een control `date`, misschien handig hier?

Reply:
Ja ziet er wel goed uit. Maar ik denk dat we in dit geval beter die string behouden. Backend geeft ons zo'n date string terug dus ik map dat liever 1 op 1